app/views/ops/_schedule_form_filter.html.haml

```diff
@@ -2,7 +2,19 @@
   .div
     .div
       -# %h4 {{buildLegend()}}
-      %div{"ng-switch" => "", "on" => "scheduleModel.action_typ"}
+      %div{"ng-switch" => "", "on" => "automateRequest()"}
```

Comment:
This should break all the other kinds... `automateRequest()` returns true or false,
but all those `ng-switch-when` assume the actual type (`vm`, `vm_check_compliance`, ...).

So.. none of the filter_typ selects will be visible now.